crates/bevy_render/src/camera/projection.rs

```diff
@@ -266,3 +266,84 @@ impl Default for OrthographicProjection {
         }
     }
 }
+
+impl OrthographicProjection {
+    /// Create a new `OrthographicProjection` with default values.
+    ///
+    /// Use its other methods to modify its properties, or modify them after creation if `mut`.
+    ///
+    /// ```
+    /// use bevy_render::camera::{OrthographicProjection, ScalingMode};
+    /// let projection = OrthographicProjection::new()
+    ///     .scale(1.0)
+    ///     .clipping_plane(0.0, 1000.0)
+    ///     .viewport_origin(0.5, 0.5)
+    ///     .scaling_mode(ScalingMode::WindowSize(1.0));
+    /// ```
+    pub fn new() -> Self {
+        OrthographicProjection {
+            left: 0.0,
+            right: 0.0,
+            bottom: 0.0,
+            top: 0.0,
+            scale: 1.0,
+            near: 0.0,
+            far: 1000.0,
+            viewport_origin: (0.5, 0.5),
+            scaling_mode: ScalingMode::WindowSize(1.0),
+        }
+    }
+
+    /// Set the scale of the projection.
```

Comment:
Is there a way to link to `scale`'s documentation here? [\`scale\`] doesn't work for fields.

Reply:
You can do [`scale`](Foo::scale) and that should work.